readme/linux.md

```diff
@@ -57,3 +57,8 @@ $ sudo apt-get install -y default-jdk
 ```bash
 $ sudo apt-get install -y lua5.3 lua-socket lua-sec
 ```
+
+#### Assembleur x86 && x64
```

Comment:
Could you replace with `x86 and x64 Assembly` or something like that?

Reply:
Translation done